src/table/table-role/grid-navigation.tsx

```diff
@@ -100,13 +91,28 @@ class GridNavigationHelper {
     this._pageSize = pageSize;
   }
 
+  public refresh() {
+    if (this._table) {
+      const cellSuppressed = this.focusedCell ? this.isSuppressed(this.focusedCell.element) : false;
```

Comment:
Should this run after the focusedCell needs to be updated?

Reply:
That makes sense, will update